lib/ronin/cli/commands/ip.rb

```diff
@@ -226,18 +239,32 @@ def process_value(ip)
         #
         def format_ip(ip)
           if options[:hex]
-            "0x%x" % ip.to_i
+            "0x%x" % ip.to_i 
+          elsif options[:hex_octet]
+            octets = ip.to_s.split(".").map!(&:to_i)
+            octets.map { |octet| octet.to_s(16) }.join(".")
```

Comment:
These two `.map` calls can be combined into one.

Reply:
I initially had it that way but I was afraid it was unreadable.  I'll go ahead and put them back together.